kinto_http/cli_utils.py

```diff
@@ -58,6 +63,14 @@ def add_parser_options(parser=None,
                         help='BasicAuth token:my-secret',
                         type=str, default=default_auth, action=AuthAction)
 
+    parser.add_argument('--bearer-type',
+                        help='Bearer token authorization realm. Default: Bearer',
+                        type=str, default=default_bearer_type)
```

Comment:
Can't it be obtained in capabilities? I would specify `--auth-provider`  instead ? 

```
$ curl -s https://kinto.dev.mozaws.net/v1/ | jq .capabilities.openid.providers 
[
  {
    "issuer": "https://auth.mozilla.auth0.com/",
    "userinfo_endpoint": "https://auth.mozilla.auth0.com/userinfo",
    "header_type": "Bearer",
    "name": "auth0",
    "auth_path": "/openid/auth0/login",
    "client_id": "XNmXEZhGfNaYltbCKustGunTbH0r8Gkp"
  }
]
````
```

Reply:
For the openid provider it seems to work, but how do you work with plugins that expose multiple authentication policies?

Also this means adding a call to the server before to validate this. I would be :-1: on this one. A client must be able to know what the protocol is.

Reply:
I guess we could explain in the documentation how a plugin should behave taking the example of the openid.

We could also in case of a 401 fetch the /v1/ page to list all possible realms? However we should keep a way to express it. Especially when providing multiple authentication policies.

Reply:
Yes I agree. I forgot that kinto-facebook is not a normal openid provider (which is sad BTW)